fossunited/foss_hackathon/doctype/foss_hackathon_participant/foss_hackathon_participant.py

```
@@ -82,22 +99,81 @@ def handle_add_to_email_group(self):
+            try:
+                handle_email_group_subscription(
+                    emails=[self.email],
+                    chapter=event_doc.chapter,
+                    event=target_localhost,
+                    event_type="Event Participants",
+                    custom_group_title=f"{status}-{target_localhost}-Localhost",
+                    subscribe_to_chapter=False,
+                    subscribe_to_event=False,
+                    document_type_event=HACKATHON_LOCALHOST,
+                )
+            except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)